cpu/stellaris_common/license.html

```diff
@@ -0,0 +1,364 @@
+<html>
```

Comment:
To me it seems that this license only applies to binary software and its documentation provided by TI to work (flash, configure, whatever) the platform, but not to the source code itself.